[MXNET-1232] fix demo and add Eclipse support (#13979)
apache/incubator-mxnet: scala-package/mxnet-demo/scala-demo/README.md

# MXNet Scala Sample Project
This is an project created to use Maven-published Scala package with two Scala examples.
## Setup

> User are required to use maven to build the package, by simply running

Reviewer: ```suggestion
You are required to use maven to build the package, by running the following:
```
Again, is it really a requirement or one route to do things?

export SCALA_TEST_ON_GPU=1
```
## Clean up

> Clean up for Maven package is simple, you can just run

Reviewer: ```suggestion
To clean up a Maven package, run the following:
```
